src/test/java/net/minecraftforge/debug/CrashCallableTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug;
+
+import net.minecraftforge.fml.CrashReportCallables;
+import net.minecraftforge.fml.common.Mod;
+
+/**
+ * Test features and guards of crash-callables.
+ * <ul>
+ * <li>The "AlwaysActiveCrashCallable" must always be printed to the crash report</li>
+ * <li>The "ToggleableCrashCallable" must only be printed to the crash report when the {@code ENABLED} flag is {@code true}</li>
+ * <li>The "BadContentCrashCallable" must always print "BadContentCrashCallable: ERR" to the crash report and
+ * print an exception to the log (relies on a try-catch in vanilla code)</li>
+ * <li>The "BadFlagCrashCallable" must never print to the crash report and must print an exception to the log</li>
+ * </ul>
+ */
+@Mod("crash_callable_test")
+public class CrashCallableTest
```

Comment:
This could mention the `F3 + C` shortcut which force-crashes the game when held for 10 seconds, for ease of testing. I'll leave it up to you whether to add it or not.

Reply:
Added a comment mentioning the shortcut